marlowe/test/Spec/Marlowe/Common.hs

```diff
@@ -260,12 +262,15 @@ shrinkAction action = case action of
 
 
 caseRelGenSized :: Int -> Integer -> Gen (Case Contract)
-caseRelGenSized s bn = Case <$> actionGenSized s <*> contractRelGenSized s bn
-
+caseRelGenSized s bn = frequency [ (9, Case <$> actionGenSized s <*> contractRelGenSized s bn)
```

Comment:
Would it be worthwhile adding a QuickCheck test to serialise and deserialise merkelized contracts, or do the existing test cases implicitly do this already?

Reply:
Yes, there is a test for serialising and deserialising contracts that are generated by this generator (I think)